.circleci/tests/nimbus-mev.json

```diff
@@ -4,13 +4,14 @@
         "cl_client_type": "nimbus"
     }],
     "network_params": {
-        "network_id": "3151908",
-        "deposit_contract_address": "0x4242424242424242424242424242424242424242",
-        "seconds_per_slot": 12,
-        "num_validator_keys_per_node": 64,
-        "preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
-        "capella_fork_epoch": 1
+        "seconds_per_slot": 12
     },
     "global_client_log_level": "info",
-    "mev_type": "full"
+    "mev_type": "full",
+    "mev_params": {
+      "mev_relay_image": "flashbots/mev-boost-relay:0.28.0a2",
```

Comment:
does it have to use these unreleased images?

Reply:
reverted.